utils/requirements.txt

```
@@ -2,3 +2,4 @@ click==7.1.2
 packaging==20.3
 pyyaml==5.4.1
 requests==2.25.1
+tomli
```

I would suggest to pin the version here:
```suggestion
tomli==2.0.0
```